Dockerfile

```
@@ -3,9 +3,9 @@ FROM golang:1.15-alpine as builder
 #ENV CGO_ENABLED=0
 WORKDIR /build
 COPY go.mod go.sum *.go ./
+COPY cmd/ ./cmd
```

```suggestion
COPY go.mod go.sum **/*.go ./
```

I think this works? Otherwise it might be easier to just do 

```suggestion
COPY . .
```